packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -984,6 +1058,64 @@ class MediaQuery extends InheritedModel<_MediaQueryAspect> {
     );
   }
 
+  /// Wraps the `child` in a [MediaQuery] with its [MediaQueryData.textScaler]
+  /// set to [TextScaler.noScaling].
+  ///
+  /// The returned widget must be inserted in a widget tree below an existing
+  /// [MediaQuery] widget.
+  ///
+  /// This can be used to prevent, for example, icon fonts from scaling as the
+  /// user adjusts the platform's text scaling value.
+  static Widget disableTextScaling({
```

Comment:
Super nit: The name `disableTextScaling` somewhat gives me the impression that it will disable text scaling for the entire widget subtree, when really it's just setting the scale to `1.0`.  Another MediaQuery widget lower in the tree would override this setting and still be able to set the scale to something else.

Is it worth explaining this in the docs?  I'm also happy to keep it as-is, just throwing that out there.

Reply:
`withNoTextScaling` ?

Reply:
I really like that!